README.md

```diff
@@ -161,6 +161,94 @@ EventTracer.info action: 'Action', message: 'Message',
 #  counter_2, 2
 ```
 
+
+### DynamoDB integration
+
+**Prerequisites:**
+- Sidekiq
+- AWS DynamoDB SDK
+
+Before using this logger, you need to define:
+```ruby
+DynamoDBConfig.config.app_name = 'guardhouse'.freeze # app name that will be sent with each log to DynamoDB
+DynamoDBConfig.config.table_name = ENV.fetch('AWS_DYNAMODB_LOGGING_TABLE', 'logs') # send logs to this DynamoDB table
```

Comment:
Could we make the example using this syntax?

```ruby
SomeClass.configure do |config|
  config.app_name = "some value"
  config.table_name = "some table name"
end
```

This allows us to override `.configure` and add validation if necessary.